app/retail/templates/mission.html

```diff
@@ -26,212 +26,356 @@
-                    <br />
-                    {% blocktrans %}<span class="text-highlight-dark-blue">DISTRIBUTED MOVEMENT</span> AIMED TO <span class="text-highlight-dark-blue">DRIVE</span>{% endblocktrans %}
-                    <br />
-                    {% blocktrans %}<span class="text-highlight-dark-blue">CHANGE</span> AND <span class="text-highlight-pink">GROW OPEN SOURCE</span>{% endblocktrans %}
-                </div>
-            </div>
+<div class="container-fluid" style="background: white;" id="mission_hero">
+  <div class="row p-5 p-5 animatedParent">
+    <div class="col-12 col-lg-10 offset-md-0 offset-lg-1 d-flex flex-sm-row flex-column text-center animated fadeInLeftShort">
+      <div class="text">
+        <div class="font-bigger-4 header-text">
+          {% blocktrans %}
+          GITCOIN IS A
+          <span class="text-highlight-dark-blue">MISSION</span> AND
+          <span class="text-highlight-dark-blue">VALUE-ORIENTED</span>
+          {% endblocktrans %}
+          <br />
+          {% blocktrans %}
+          <span class="text-highlight-dark-blue">
+            DISTRIBUTED MOVEMENT
+          </span> AIMED TO
+          <span class="text-highlight-dark-blue">DRIVE</span>
+          {% endblocktrans %}
+          <br />
+          {% blocktrans %}
+          <span class="text-highlight-dark-blue"> CHANGE </span> AND
+          <span class="text-highlight-pink">GROW OPEN SOURCE</span>
+          {% endblocktrans %}
         </div>
+      </div>
     </div>
+  </div>
 </div>
 
-
-<div class="container-fluid" style="background-color: #eef4ee;" id="we_want">
-    <div class="row">
-        <div class="col pb-4">
-            <h4 class="text-left text-uppercase">
-                {% blocktrans %}<span class="text-highlight-green">We</span> Want{% endblocktrans %}
-            </h4>
-        </div>
+<div class="container-fluid" style="background-color: #f2f6f9;" id="we_want">
+  <div class="row">
+    <div class="col pb-4">
+      <h4 class="text-left text-uppercase">
+        {% blocktrans %}
+        <span class="text-highlight-green">We</span> Want
+        {% endblocktrans %}
+      </h4>
     </div>
-
-    <div class="row mb-5 animatedParent">
-        <div class="col d-flex flex-column align-items-center animated fadeInLeftShort">
-            <div class="mission-img-wrapper d-flex justify-content-center align-items-center">
-                <img src="{% static "v2/images/mission/want/financial.png" %}" />
-            </div>
+  </div>
+
+  <div class="row animatedParent we-want">
+    <div class="col-md-12 col-lg-8 col-xl-6 animated fadeInLeftShort">
+      <div class="row mb-5">
+        <div class="col-md-10 col-lg-8 d-flex align-items-center">
+          <img
+            class="icon"
+            src="{% static "v2/images/mission/want/rocket_man.svg" %}"
+          />
+          <span class="icon-text">
+            We want to build a world in which anyone has the financial
```

Comment:
Can we wrap these remaining strings in `trans` and `blocktrans` where appropriate?

Reply:
@mbeacom done.. wrapped all p/span strings